client/app/visualizations/box-plot/renderer.less

```diff
@@ -0,0 +1,4 @@
+.box-plot-deprecated-visualization-container {
+  overflow: hidden;
+  height: 400px;
```

Comment:
Looks like the original is `500px` so this renders smaller.

Reply:
I decided to set it to `400px` because other visualizations use this size as well. It's only for query page, on dashboard pages now it will fit container (like chart, counter, word cloud, etc. - see screenshots). I think it's not an issue at all.

Reply:
Don't you think it looks a bit too small when in query view mode.
<img width="1440" alt="Screen Shot 2019-07-03 at 20 20 42" src="https://user-images.githubusercontent.com/486954/60611889-15710e80-9dd0-11e9-8da3-e90a692a7812.png">

Reply:
(Actually, the best option is actually to make this query view resize observable)

Reply:
(different scope)